assets/sass/components/_issue.scss

```diff
@@ -33,7 +35,7 @@ body.issue {
       font-size: 3rem;
       font-weight: 700;
       text-align: left;
-      margin: 0 0 10px 0;
+      margin: -0.5rem 0 2rem 0;
```

Comment:
shift the header up a smidge like we did for the archive pages for better top alignment with the blue box on the left